app/views/account/dashboard/inbox.html.slim

```diff
@@ -0,0 +1,31 @@
+- title "Inbox"
+.container
+  .level
+    .level-left
+      .title Inbox
+  - if @new_messages.any? || @old_messages.any?
+    table.table.is-fullwidth.is-hoverable
+      tbody.tbody
+        - if @new_messages.any?
+          tr.tr
+            td[colspan=2]
+              .td.has-text-weight-bold.is-size-5 New messages:
+          - @new_messages.each do |message|
+            tr.tr
+              td.td
+                = message.body
+              td.td.has-text-right
+                = l message.created_at, format: :date_month_year_concise
+        - if @old_messages.any?
+          tr.tr
+            td[colspan=2]
+              .td.has-text-weight-bold.is-size-5 Old messages:
+          - @old_messages.each do |message|

```

Comment:
do we have difference between templates for old messages and new messages? if no, you can move them to shared partial